moda-contracts/test/Catalog.t.sol

```diff
@@ -23,6 +25,8 @@ contract CatalogTest is Test {
 
     function setUp() public {
         modaRegistry = new ModaRegistry();
+        management = new Management();
+        modaRegistry.setManagement(address(management));
```

Comment:
Let's add management, treasury, and splits factory args to the constructor so we don't have to do extra transactions like this